content/en/blog/_posts/2024-secret-sync-controller-alpha.md

```diff
@@ -0,0 +1,56 @@
+
+3. Security: The Secret Sync Controller (SSC) has limited permissions and leverages the latest Kubernetes security features (full list below) so that cluster admins do not need to configure and limit permissions themselves.  
+
+## How to use the Secret Sync Controller 
+The Secret Sync Controller (SSC) must be used alongside a [secret store Cloud Provider](https://secrets-store-csi-driver.sigs.k8s.io/getting-started/installation.html) that implements the Secret Store CSI Driver provider interface to provide an end-to-end solution for synchronizing secrets from the cloud secret store to the Kubernetes secret store.  
+
+1. Install the secret sync controller [LINK TBD] via a helm chart or YAML manifests alongside a secret store provider. If using the helm chart, at install time users can configure parameters such as sync intervals and resource limitations. 
+
+2. Configure an identity that has permission to read secrets from the cloud provider.  
+
+3. Configure the cloud provider with a secret provider class CRD.  
+
+4. Configure the Secret Sync Controller with a secret sync CRD that contains the associated cloud provider CRD, name of the secret in the cluster, target namespace, and the Kubernetes service account that has federated identity access to the Cloud Provider secret.  
+
+5. Apply the CRDs in the cluster and secrets will automatically begin syncing.  
+
+## Security Features 
+By default, the Secret Sync Controller (SSC) enables the following security features: 
+
+1. Isolation of Functionality: The SSC splits out the functionality of syncing secrets from a cloud provider (via a secret sync Provider) and storing the secrets in a cluster (the SSC). Each component can have much more limited permissions, minimizing the risk of a cluster compromise.  
+
+2. Limited Permissions via Validating Admission Policies: The SSC is a privileged resource that should be in the kube-system namespace (or a similarly tightly protected namespace). The SSC leverages validating admission policies to limit its capabilities, ensuring it cannot perform operations such as deleting secrets or editing secrets beyond its designated scope (defined via labels and annotations). Validating admission policies will be GA and enabled by default starting with Kubernetes 1.30. 
+
+3. Cloud Provider vs Workload Configuration: Configuration of the secrets to be synced to the cluster are split into two layers – the cloud provider configuration and the workload specific configuration. This allows a cloud provider admin to create and define the cloud provider secret storage and cloud identity that has access to secrets. The workload developer can then define how the secret is stored in the cluster (name, namespace, etc) and which local Kubernetes identity has access to the secret.  
+
+4. Namespace Boundaries: The SSC leverages the Kubernetes namespace security boundary to isolate secrets so that only pods in a namespace have access to secrets in that namespace. 
```

Comment:
```suggestion
4. Namespace Boundaries: The SSC leverages the Kubernetes namespace security boundary to isolate Secrets so that only Pods in a namespace have access to Secrets in that namespace.
```

Do we have any comment to make about ReferenceGrant (the upcoming mechanism for cross-namespace references)?